File: app/src/main/java/com/ramtomka/optivida/LoginActivity.java
package com.ramtomka.optivida;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class LoginActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);

        Button btnLogin = findViewById(R.id.btnLogin);

        // EXPRESION LAMBDA
        btnLogin.setOnClickListener(v -> {
            Intent intent = new Intent(getApplicationContext(), MainActivity.class);
            startActivity(intent);
        });
    }
}

File: app/src/main/java/com/ramtomka/optivida/LoginActivity.kt
package com.ramtomka.optivida

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.TextView

class LoginActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)

        val btnLogin = findViewById<Button>(R.id.btn)

        val btnOlvidar = findViewById<TextView>(R.id.OlvidarPass)

        val btnRegistrar = findViewById<TextView>(R.id.registrar)

        btnLogin.setOnClickListener{
            //INICIAR OTRA ACTIVIDAD

            val intent = Intent(this,MainActivity::class.java)
            finish()
            startActivity(intent)

        }

        btnOlvidar.setOnClickListener {
            val intent = Intent(this, CambiarPass::class.java)
            startActivity(intent)
            finish()
        }

        btnRegistrar.setOnClickListener {
            val intent = Intent(this, Registrar::class.java)
            startActivity(intent)
            finish()
        }
    }
}